half_asec_half_desc.cpp

```diff
@@ -0,0 +1,116 @@
+	
+/*8.Given a linked list which is in pattern like combination of ascending and descending. Sort it.(First all ascending numbers occur ,next all descending nums )
+Note : min of descending part is greater than max of ascending 
+Ex: Input 1->2->7->20->15->10
+Output 1->2->7->10->15->20*/
```

Comment:
Sir, clearly it was given that half of it is ascending order but in descending order we use to make them sort